src/plugins/chart_expressions/expression_legacy_metric/public/components/metric_value.tsx

```diff
@@ -61,6 +61,7 @@ export const MetricVisValue = (props: MetricVisValueProps) => {
           data-test-subj="metric_label"
           style={{
             ...(labelConfig.style.spec as CSSProperties),
+            fontSize: '24px',
```

Comment:
Enforce a specific font size for a single target app does not look like the right solution as this is a generic renderer function.